### PR TITLE
add stageflix from suckcoder

### PR DESCRIPTION
**Pull request by suckcoder**
### New Stylesheet Submissions:

stageflix

- [x] Have you followed the guidelines in our Contributing document?
- [x] Have you checked to ensure there aren't other files using the same name as yours?
- [x] Have you filled in at least the required data for `title`, `author`, and `stylesheet`?
- [x] Is the value of `stylesheet` a full URL that is publicly accessible and renders a compiled CSS file?

i am beginer for github apologize for the mistake.

### CODE REVIEW

File: src/_data/styles/stageflix.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Stageflix",
+  "author": "Suckcoder",
+  "stylesheet": "https://codepen.io/suckcoder/pen/KKVbObL.css",
```
Comment (5t3ph) [inline src/_data/styles/stageflix.json:4]:
```suggestion
  "stylesheet": "https://codepen.io/suckcoder/pen/KKVbObL.css"
```

Remove the comma and then it should work!

**Comment by 5t3ph** [comment]
Hi @suckcoder! No worries, I resolved the small issue with the `json` formatting.

Here's the preview link: https://deploy-preview-20--moderncss-styles.netlify.app/styles/stageflix/

I really like what you've created! Just a few things to address:
- Missing styles for `.skip-link` (the very first item in the page, you can see how it's done on stylestage.dev by pressing `tab` from the address bar at which point it appears)
- Can you confirm that the graphics you used are open source or licensed to be used non-commercially? (as in they weren't just taken from another brand's site) If they were ones you found within CodePen's assets then that's perfect!
- The "toast" illustration is awesome but it's hard to read the link text - when I adjusted the background-position that helped: `background-position: center -75px;`

Let me know when you've made these adjustments and I'll redeploy the preview!

**Comment by suckcoder** [comment]
@5t3ph 

Thanks you for your suggestion, all graphics on stagefix are open source and can be use for non-commercially and i already adjust the background position of "toast" illustration for`.skip-link` style is done (sorry for my english language).

love this project :)

**Comment by 5t3ph** [comment]
@suckcoder great, I'll get it merged! Nice job on this!

I'd like to invite you to subscribe to updates to keep informed of new features: https://stylestage.dev/subscribe/

**Pull request merged**